configs/datamodule/configuration/template_configuration.yaml

```diff
@@ -0,0 +1,98 @@
+general:
```

Comment:
Do we need all 4 of these configurations? Could you add alittle readme saying what they are and why?